tests/e2e/forms/immunization-visit-form.standard-wdio-spec.js

```diff
@@ -0,0 +1,166 @@
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.CHOLERA_VACCINE, 'CH');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HEPATITIS_A_VACCINE, 'HA');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HEPATITIS_B_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.HPV_VACCINE, 'HPV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.FLU_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.JAP_ENCEPHALITIS_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MENINGOCOCCAL_VACCINE, 'MN');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MMR_VACCINE, 'MMR');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.MMRV_VACCINE, 'MMRV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.POLIO_VACCINE, 'PV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.PENTAVALENT_VACCINE, 'DT');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.DPT_BOOSTER_VACCINE, 'DPT');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.PNEUMOCOCCAL_VACCINE, 'PCV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.ROTAVIRUS_VACCINE, 'RV');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.TYPHOID_VACCINE, 'TY');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.VITAMIN_A_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.selectAppliedVaccines(immunizationVisitForm.YELLOW_FEVER_VACCINE, 'yes');
+        await genericForm.nextPage();
+        await immunizationVisitForm.addNotes();
+        await genericForm.nextPage();
+
+        expect(await immunizationVisitForm.getPatientNameSummaryPage()).to.equal(babyName);
+        expect(await immunizationVisitForm.getAppliedVaccines()).to.equal(await immunizationVisitForm.getAppliedVaccinesSummary() - 1);
+        expect(await immunizationVisitForm.getFollowUpSMS()).to.include(babyName && await immunizationVisitForm.getNotes());
```

Comment:
The baby name is `Baby1`, so I modified the assertion to look for `raul`, but the test still passed. It should have failed because the name isn't the expected. 
```
expect(await immunizationVisitForm.getFollowUpSMS()).to.include('raul' && babyMedicID && await immunizationVisitForm.getNotes());
```

The reason is because in Javascript this statement:
```
babyName && babyMedicID && await immunizationVisitForm.getNotes()
```
It's the equivalent of:
```
if (babyName !== undefined && babyMedicID !== undefined) {
   await immunizationVisitForm.getNotes();
}
```
An easy test in the browser's console would be:
```
"Hello miss, can I have mac & cheese please?".includes("miss" && "mac & cheese"); // returns true
"Hello miss, can I have mac & cheese please?".includes("Cleo" && "mac & cheese"); // returns true
```
I way to fix it is checking if the message include each of these separately:
```
expect(message).to.include(babyName);
expect(message).to.include(babyID);
... etc
```

Reply:
Oh wow! I didn't know that and I didn't test it. Thank you!!